sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -110,6 +112,49 @@ object ScalaJSPlugin extends AutoPlugin {
       new PhantomJSEnv(executable, args, env, autoExit, loader)
     }
 
+    /**
+     *  Creates a [[sbt.Def.Initialize Def.Initialize]] for a SeleniumJSEnv. Use
+     *  this to explicitly specify in your build that you would like to run with
+     *  Selenium:
+     *
+     *  {{{
+     *  // Using Firefox
+     *  jsEnv := SeleniumJSEnv().value
+     *  jsEnv := SeleniumJSEnv(keepAlive = true).value
+     *
+     *  // Using Google Chrome
+     *  jsEnv := SeleniumJSEnv(SeleniumChrome("/../chromedriver")).value
+     *  jsEnv := SeleniumJSEnv(SeleniumChrome("/../chromedriver"), keepAlive = true).value
+     *  }}}
+     *
+     *  Note that the resulting [[sbt.Def.Setting Setting]] is not scoped at
+     *  all, but must be scoped in a project that has the ScalaJSPlugin enabled
+     *  to work properly.
+     *  Therefore, either put the upper line in your project settings (common
+     *  case) or scope it manually, using
+     *  [[sbt.ProjectExtra.inScope[* Project.inScope]].
+     */
+    def SeleniumJSEnv(
+       browser: SeleniumBrowser = SeleniumFirefox(),
+       keepAlive: Boolean = false
+    ): Def.Initialize[Task[SeleniumJSEnv]] = Def.task {
+      new SeleniumJSEnv(browser, keepAlive)
+    }
+
+    /** Firefox browser configuration for SeleniumJSEnv(browser). */
+    def SeleniumFirefox(): SeleniumBrowser =
+      selenium.Firefox
+
+    /** Google Chrome browser configuration for SeleniumJSEnv(browser).
+     *
+     *  @param chromedriverPath Path to the Selenium chrome driver.
+     *    Can be downloaded from http://chromedriver.storage.googleapis.com/index.html
+     */
+    def SeleniumChrome(chromedriverPath: String): SeleniumBrowser = {
+      System.setProperty("webdriver.chrome.driver", chromedriverPath)
```

Comment:
I could just remove the parameter and let the user add it to the `javaOptions` as a `-D`.